projects/open-dex/index.js

```diff
@@ -0,0 +1,29 @@
+const { getUniTVL, sumUnknownTokens } = require("../helper/unknownTokens");
+const sntNova = "0x657a66332a65b535da6c5d67b8cd1d410c161a08";
+const odxToken = "0xe36118ccfa51e4caf750dfaeaf5c6cf250759acb";
+const qsrToken = "0x356c044B99e9378C1B28A1cAb2F95Cd65E877F33";
+const nUSD = "0x1F5396f254EE25377A5C1b9c6BfF5f44e9294fFF"
+const dexFactory = "0x9550b0c83AD5a58898cD4267987Af67e7E52bF55";
```

Comment:
this is fusion dex factory address, what is open dex factory address?

Reply:
it routes it via aggregation, uses fusion factory

Reply:
sorry, not sure if you are realizing it, current PR is counting all of fusion tvl as yours

if your UI supports creating pools using their factory, the new tvl is still theirs.

Reply:
yes, its an aggregator, i thought you would add it as you do with yield optimizers or staking protocols, with the double-counting toggle and warning, do i need to incorporate that into my adaptor?

Reply:
sorry, yield optimizer have their own contracts, we dont count count aggregator unless they have their own tvl